graph/assemblygraph.h

```diff
@@ -191,10 +198,60 @@ class AssemblyGraph : public QObject
     bool attemptToLoadSequencesFromFasta();
     long long getTotalLengthOrphanedNodes() const;
     bool useLinearLayout() const;
+
+    // FIXME: This belongs to UI controller!
     bool nodeHasBlastHit(const DeBruijnNode *node) const;
     bool nodeOrReverseComplementHasBlastHit(const DeBruijnNode *node) const;
     const std::vector<std::shared_ptr<BlastHit>> &getBlastHits(const DeBruijnNode *node) const;
 
+    bool hasCustomColour(const DeBruijnNode* node) const {
```

Comment:
Maybe move it to cpp for consistency with other code? I don't think it will be noticeably slower.

Reply:
Done